tests/h/views/api_auth_test.py

```diff
@@ -24,7 +24,7 @@
 from h.views import api_auth as views
 
 
-@pytest.mark.usefixtures('routes', 'oauth_validator', 'user_svc')
+@pytest.mark.usefixtures('routes', 'user_svc', 'oauth_validator')
```

Comment:
Sort of. The `oauth_validator` fixture calls the service factory which loads the user service. But I've changed it so that the `oauth_validator` fixture loads the `user_svc` fixture on its own and that the order of fixtures here is irrelevant.